core/domain/draft_upgrade_services.py

```diff
@@ -0,0 +1,82 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Commands that can be used to upgrade draft to newer Exploration versions."""
+import logging
+
+from core.domain import exp_domain
+from core.platform import models
+
+(exp_models, feedback_models, user_models) = models.Registry.import_models([
+    models.NAMES.exploration, models.NAMES.feedback, models.NAMES.user
+])
+
+
+def try_upgrade_from_version_to_version(
+        draft_change_list, from_version, to_version, exp_id):
+    """Try upgrading an ExplorationChange domain object by calling convert
+    functions for a list of commits.
+
+    For now, this handles only the scenario where commits_list contains a
```

Comment:
Do you mean: "... where each item in the change list contains a single commit that ..." ?

Reply:
Right, thanks! Will update the comment as suggested.

Reply:
Updated the comment PTAL